nixos/dokuwiki: caddy support

onny opened the pull request:
<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!
List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://nixos.org/manual/nixpkgs/unstable/#chap-reviewing-contributions
-->

###### Motivation for this change

To be able to combine DokuWiki with other modules that are using Caddy. This concept is unfluenced by [this pull request](https://github.com/NixOS/nixpkgs/pull/84446).

###### Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [x] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

thiagokokada committed:
nixos/noisetorch: init

NoiseTorch needs setcap set to 'cap_sys_resource=+ep' to work correctly
accordingly to the README.md:

https://github.com/lawl/NoiseTorch#download--install

So this PR adds it.

r-ryantm committed:
coordgenlibs: 2.0.2 -> 2.0.3

r-ryantm committed:
onedrive: 2.4.10 -> 2.4.11

reedrw committed:
r2mod_cli: 1.0.7 -> 1.2.0

jojosch committed:
jellyfin-web: init at 10.7.2

jojosch committed:
jellyfin: build from source

r-ryantm committed:
libofx: 0.10.1 -> 0.10.2

r-ryantm committed:
bitcoin-unlimited: 1.9.0.1 -> 1.9.1.1

r-ryantm committed:
freetds: 1.2.18 -> 1.2.21

r-ryantm committed:
hdparm: 9.60 -> 9.61

Commit:
mix2nix: init at 0.1.2

Commit:
use `teams.beam.members` to add beam team to maintainers

Commit:
fix typo in maintainers list

jojosch committed:
jellyfin-web: 10.7.2 -> 10.7.5

jojosch committed:
jellyfin: 10.7.2 -> 10.7.5

jnetod committed:
valentina: downgrade qt to 5.12

bbjubjub2494 committed:
ghc-filesystem: init at 1.5.4

bbjubjub2494 committed:
pythonPackages.chiapos: init at 1.0.1

bbjubjub2494 committed:
pythonPackages.chiavdf: init at 1.0.1

bbjubjub2494 committed:
pythonPackages.chiabip158: init at 1.0

bbjubjub2494 committed:
chia: init at 1.1.5

bbjubjub2494 committed:
pythonPackages.keyrings-cryptfile: init at 1.3.4

bbjubjub2494 committed:
pythonPackages.blspy: init at 1.0.2

bbjubjub2494 committed:
pythonPackages.clvm-rs: init at 0.1.7

bbjubjub2494 committed:
maintainers: add chia team

bbjubjub2494 committed:
pythonPackages.clvm: init at 0.9.6

bbjubjub2494 committed:
pythonPackages.clvm-tools: init at 0.4.3

bbjubjub2494 committed:
pythonPackages.concurrent-log-handler: init at 0.9.19

fitzgibbon committed:
gnome-inform7: init at unstable-2021-04-06

onny committed:
nixos/dokuwiki: caddy support

jraygauthier committed:
pythonnet: 2.4.0 -> 2.5.2

 -  No longer broken.
 -  Updated mono dependency to mono5.
 -  Fix / improve packaging.

polygon committed:
blobfuse: Bump version, separately build cpplite

* Bump version to most recent 1.3.7 release
* Build scripts have changed, cpplite (azure-storage-lite) is now
  built separately
* Upstream they use build.sh to copy resulting static libraries to
  locations for blobfuse build, I put cpplite in a separate build
* adls library from cpplite was not installed and hence, not found
  by main blockfuse build, added a patch to install adls library, too

1000101 commented:
@GrahamcOfBorg test dokuwiki

1000101 commented on nixos/modules/services/web-apps/dokuwiki.nix:
```diff
@@ -43,346 +49,404 @@ let
+          "~ /(conf/|bin/|inc/|install.php)" = {
+            extraConfig = "deny all;";
+          };
+          
```
```suggestion

```

1000101 commented on nixos/modules/services/web-apps/dokuwiki.nix:
```diff
@@ -43,346 +49,404 @@ let
               fastcgi_param SCRIPT_FILENAME $document_root$fastcgi_script_name;
               fastcgi_param REDIRECT_STATUS 200;
               fastcgi_pass unix:${config.services.phpfpm.pools."dokuwiki-${hostName}".socket};
               ${lib.optionalString (cfg.nginx.addSSL || cfg.nginx.forceSSL || cfg.nginx.onlySSL || cfg.nginx.enableACME) "fastcgi_param HTTPS on;"}
```
Have you tried to run the tests manually? Both manual and ofborg/automated tests [fail](https://github.com/NixOS/nixpkgs/pull/122883/checks?check_run_id=2585723073) on this line

onny replied:
Thank you for your feedback! I use ``nixpkgs-review wip`` to test the commits. Is there more I can do?

1000101 replied:
Hi @onny. Yes, there are tests that need to pass (and be updated if they need to be). It would also be great if you could try it in a live environment (I used to test it on a real VM to test more complex scenarios).

Anyway, to run at least the local tests, you should run `nix-build -A  nixosTests.dokuwiki` from your `nixpkgs` repo or from `nixpkgs` directory when doing `nixpkgs-review wip`.

E.g.:
```
nixpkgs $ nix-review pr 122883
.cache/nixpkgs-review/pr-122883-1$ cd nixpkgs
.cache/nixpkgs-review/pr-122883-1/nixpkgs$ nix-build -A  nixosTests.dokuwiki
```

stephank committed:
xvfb-run: mark as linux-only

Commit:
nixos: convert netatalk to settings-style configuration

Also, set StateDirectory in systemd.….serviceConfig.

zeri42 committed:
zod: 2011-03-18 -> 2011-09-06

- cleanup Derivation
- add map_editor and zod_launcher
- game is playable now (appears broken/orphaned in current version)
- fix comment about version
- add enableParallelBuilding=true;
- change source url to use mirror://

onny commented:
Oh I have to add, this pull request depends on https://github.com/NixOS/nixpkgs/pull/122647 , but hope it will be merged soon :)

onny commented:
> Oh I have to add, this pull request depends on #122647 , but hope it will be merged soon :)

Group and user support merged into Caddy now :)

r-ryantm committed:
agi: 1.1.0-dev-20210513 -> 1.1.0-dev-20210514

r-ryantm committed:
argo: 3.0.3 -> 3.0.4

r-ryantm committed:
earthly: 0.5.12 -> 0.5.13

1000101 commented:
@GrahamcOfBorg test dokuwiki

1000101 commented:
Result of `nixpkgs-review pr 122883` [1](https://github.com/Mic92/nixpkgs-review)

1000101 commented:
The tests are still failing/not evaluating with the same error.

fabaff committed:
python3Packages.hatasmota: 0.2.12 -> 0.2.13

Commit:
bump to newest mix2nix version and move elixir to nativeBuildInputs

Jaculabilis committed:
nixos/nebula: Add release notes

r-ryantm committed:
dnsproxy: 0.37.3 -> 0.37.4

r-ryantm committed:
i2pd: 2.37.0 -> 2.38.0

Ma27 committed:
clickhouse: 20.11.4.13-stable -> 21.3.11.5-lts

Failing Hydra build: https://hydra.nixos.org/build/143269865
ZHF #122042

r-ryantm committed:
gensio: 2.2.4 -> 2.2.5

savannidgerinel committed:
_1password-gui: 8.0.33-52 -> 8.0.34

This includes an overhaul of the install script, using patchelf to set
the interpreter and the libraries. This reflects a lot of what I've
learned about electron + nixos over the last few months.

This change is motivated by the fact that 1Password's version of
electron changed, probably leading to crashes that I've been seeing when
I try to update the Nix derivation to the latest versions.

Co-authored-by: Daniël de Kok <me@github.danieldk.eu>

dotlambda committed:
python3Packages.ytmusicapi: 0.16.0 -> 0.17.1

https://github.com/sigma67/ytmusicapi/releases/tag/0.17.0
https://github.com/sigma67/ytmusicapi/releases/tag/0.17.1

dotlambda committed:
spotdl: disable tests

They require networking after the last ytmusicapi update.

fabianhjr committed:
metals: 0.10.2 → 0.10.3

zeri42 committed:
zod: add scaling factor

 - add patch that allows to resize the zod_launcher window at compile
 time with `scalingFactor` option

r-ryantm committed:
liblinphone: 4.5.1 -> 4.5.15

pschyska committed:
nixos/nsd: make nsd-checkconf work when configuration contains keys

wnklmnn committed:
gtk-sharp: fix build

peterhoeg committed:
openmw: ffmpeg_3 -> ffmpeg

happysalada committed:
ion: unstable-2020-03-22 -> unstable-2020-05-10

fix darwin build

paperdigits committed:
wlc: init at 1.11

r-ryantm committed:
cava: 0.7.3 -> 0.7.4

r-ryantm committed:
cicero-tui: 0.2.0 -> 0.2.1

r-ryantm committed:
codeql: 2.5.4 -> 2.5.5

r-ryantm committed:
esbuild: 0.11.23 -> 0.12.0

r-ryantm committed:
flyctl: 0.0.217 -> 0.0.220

r-ryantm committed:
helmfile: 0.139.3 -> 0.139.6

r-ryantm committed:
helmsman: 3.6.10 -> 3.6.11

peterhoeg committed:
openmw-tes3mp: unstable-2019-11-19 -> unstable-2020-08-07

Lassulus committed:
solanum: 2020-12-14 -> 2021-04-27

moni-dz committed:
linuxPackages.rtl8821au: switch to a more maintained fork

moni-dz committed:
linuxPackages.rtl8821ce: mark broken for aarch64

Co-authored-by: Jonathan Ringer <jonringer@users.noreply.github.com>

rhoriguchi committed:
gnomeExtensions.fuzzy-app-search: 4 -> 4.0.1

Rakesh4G committed:
pythonPackages.fastpair : Updated version and pytest enabled

chivay committed:
pythonPackages.karton-yaramatcher: 1.1.0 -> 1.1.1

r-ryantm committed:
clojure: 1.10.3.822 -> 1.10.3.839

chivay committed:
pythonPackages.karton-autoit-ripper: 1.0.0 -> 1.0.1

zowoq committed:
gh: 1.10.0 -> 1.10.1

https://github.com/cli/cli/releases/tag/v1.10.1

fabaff committed:
Merge pull request #123662 from chivay/bump-karton-asciimagic

pythonPackages.karton-asciimagic: 1.0.0 -> 1.0.1

fabaff committed:
Merge pull request #123661 from chivay/bump-classifier

pythonPackages.karton-classifier: 1.0.0 -> 1.1.0

fabaff committed:
Merge pull request #123670 from chivay/bump-config-extractor

pythonPackages.karton-config-extractor: 2.0.0 -> 2.0.1

markuskowa committed:
ucx: 1.10.0 -> 1.10.1

jtojnar committed:
Merge pull request #123629 from dasj19/adwaita-qt-update

adwaita-qt: 1.1.4 -> 1.3.0

primeos committed:
Merge pull request #123609 from berbiche/cagebreak-use-waylands-utils-in-test

nixos/tests/cagebreak: use wayland-info instead of wallutils

fabaff committed:
python3Packages.zeroconf: 0.30.0 -> 0.31.0 (#123567)

mweinelt committed:
python3Packages.coqpit: init at 0.0.6.6

mweinelt committed:
tts: 0.0.13 -> 0.0.14

https://github.com/coqui-ai/TTS/releases/tag/v0.0.14

mweinelt committed:
python3Packages.phonemizer: stop maintaining the package

mweinelt committed:
python3Packages.segments: stop maintaining the package

lheckemann committed:
matrix-synapse: add openssl to buildInputs

It should be included regardless of whether tests are run or not, and
overriding the package to disable tests would break this package up
until this change.

edude03 committed:
wayland: broken for darwin

timokau committed:
Merge pull request #122624 from collares/pari-2.13

pari: 2.11.4 -> 2.13.1, update sage test expectations

jtojnar committed:
Merge pull request #123507 from jtojnar/no-flatpak-guipkgs

nixos/flatpak: Remove `guiPackages` internal option

figsoda committed:
luaformatter: 1.3.5 -> 1.3.6

prusnak committed:
bcal: cleanup package, enable aarch64-darwin build

r-ryantm committed:
dex-oidc: 2.27.0 -> 2.28.1

r-ryantm committed:
pantheon.elementary-calculator: 1.6.0 -> 1.6.1

roberth committed:
Merge pull request #122784 from hercules-ci/nixpkgs-init-nixos-install-tools

nixos-install-tools: init

roberth committed:
Merge pull request #123591 from hercules-ci/update-podman-compose

podman-compose: 0.1.5 -> 0.2.0pre-2021-05-18

prusnak committed:
Merge pull request #123689 from prusnak/bcal

bcal: cleanup package, enable aarch64-darwin build

collares committed:
sage: fix deprecation warning from matplotlib upgrade

SuperSandro2000 committed:
Merge pull request #123520 from figsoda/macchina

macchina: 0.7.2 -> 0.8.1

SuperSandro2000 committed:
Merge pull request #123530 from figsoda/luaformatter

luaformatter: 1.3.5 -> 1.3.6

mweinelt committed:
Merge pull request #123355 from Ma27/bump-matrix-synapse

jtojnar committed:
Merge pull request #123658 from rhoriguchi/gnomeExtensions.fuzzy-app-search

gnomeExtensions.fuzzy-app-search: 4 -> 4.0.1

mweinelt committed:
Merge pull request #123641 from fabaff/bump-pysonos

python3Packages.pysonos: 0.0.48 -> 0.0.49

mweinelt committed:
Merge pull request #123456 from mweinelt/home-assistant-tests

risicle committed:
Merge pull request #123553 from DavHau/zhf-clickhouse-driver

python3Packages.clickhouse-driver: unbreak

Commit:
solc: make darwin friendly

mweinelt committed:
home-assistant: 2021.5.4 -> 2021.5.5

https://github.com/home-assistant/core/releases/tag/2021.5.5

sophrosyne97 committed:
scenebuilder: fix build error

leotaku committed:
netdata: 1.30.1 -> 1.31.0

leotaku committed:
netdata: Remove patch which is already present in new release

johnae committed:
qtwebengine: enable pipewire/webrtc

johnae committed:
qutebrowser: enable pipewire/webrtc (for wayland screensharing)

chivay committed:
fixup! pythonPackages.karton-yaramatcher: 1.1.0 -> 1.1.1

Remove yara-python substitution

mweinelt committed:
Merge pull request #123704 from mweinelt/home-assistant

home-assistant: 2021.5.4 -> 2021.5.5

fabaff committed:
Merge pull request #123269 from fabaff/locationsharinglib

python3Packages.locationsharinglib: init at 4.1.6

jonringer committed:
alacritty: 0.7.2 -> 0.8.0

stephank committed:
reproxy: 0.5.1 -> 0.6.0

ttuegel committed:
Merge pull request #123290 from eliasp/init-plasma-sdk

plasma-sdk: init at 5.21.4

stephank committed:
bacon: fix darwin build

symphorien committed:
Merge pull request #123089 from Thesola10/dduper

dduper: init at v0.04

dotlambda committed:
retroarchBare: use ffmpeg instead of ffmpeg_3 (#123514)

fabaff committed:
ipinfo: 1.1.2 -> 1.1.4

globin committed:
Merge pull request #123586 from dpausp/grafana-7.5.6

grafana: 7.5.2 -> 7.5.6

globin committed:
Merge pull request #123669 from mweinelt/prometheus

prometheus: 2.26.0 -> 2.27.1

markuskowa committed:
Merge pull request #123681 from markuskowa/upd-ucx

ucx: 1.10.0 -> 1.10.1

Flakebi committed:
opensmtpd-filter-rspamd: init at 0.1.7 (#122823)

Gabriella439 committed:
Fix description for services.kubernetes.addonManager.enable (#71448)

`mkEnableOption` already prefixes the description with
"Whether to enable"

prusnak committed:
Merge pull request #123663 from prusnak/electron-aarch64-darwin

electron: enable aarch64-darwin build (for electron 11+)

fabaff committed:
python3Packages.aioshelly: 0.6.2 -> 0.6.3

globin committed:
Merge pull request #123604 from mweinelt/esphome

esphome: 1.17.2 -> 1.18.0

risicle committed:
Merge pull request #123473 from stephank/fix-opencl-clang-darwin

opencl-clang: fix darwin build

risicle committed:
Merge pull request #123455 from jnetod/zhf-vdirsyncer

pythonPackages.vdirsyncer: fix build with python 3.9

markuskowa committed:
Merge pull request #123667 from markuskowa/upd-molcas

openmolcas: 20.10 -> 21.02

Commit:
terraform_0_15: 0.15.3 -> 0.15.4

dotlambda committed:
openjfx11: use ffmpeg instead of ffmpeg_3 (#123500)

dotlambda committed:
ffmpeg_3: mark as insecure (#123496)

lunik1 committed:
megacmd: temporarily remove ffmpeg dependency (#123711)

Build fails against ffmpeg 4.4, and ffmpeg_3 is to be marked insecure.

prusnak committed:
Merge pull request #123660 from prusnak/octant

octant: enable aarch64-darwin build

risicle committed:
python3Packages.hdbscan: disable flaky tests

r-ryantm committed:
latte-dock: 0.9.11 -> 0.9.12

mweinelt committed:
Merge pull request #123730 from fabaff/bump-aioshelly

python3Packages.aioshelly: 0.6.2 -> 0.6.3

drewrisinger committed:
python3Packages.cirq-core: init at 0.11.0

drewrisinger committed:
python3Packages.cirq-google: init at 0.11.0

drewrisinger committed:
python3Packages.cirq: 0.10.0 -> 0.11.0

Rework for the latest version of Cirq. Breaks the package into
submodules that can be used somewhat independently.

roberth committed:
ghostunnel: init at 1.5.3

A simple TLS proxy with mutual authentication support
for securing non-TLS backend applications.

risicle committed:
python3Packages.nassl: fix build

was broken when #119825 fixed openssl.bin's /bin location

roberth committed:
nixos/ghostunnel: init

marsam committed:
Merge pull request #123734 from maxeaubrey/terraform_0.15.4

terraform_0_15: 0.15.3 -> 0.15.4

marsam committed:
Merge pull request #123614 from marsam/update-pgvector

postgresqlPackages.pgvector: 0.1.2 -> 0.1.4

marsam committed:
Merge pull request #123677 from marsam/update-nodejs

nodejs: 14.16.1 -> 14.17.0, 16.1.0 -> 16.2.0

mweinelt committed:
Merge pull request #123736 from risicle/ris-hdbscan-disable-flaky-test

python3Packages.hdbscan: disable flaky test

mweinelt committed:
Merge pull request #123636 from fortuneteller2k/rtl8821cu

linuxPackages.rtl8821cu: switch to a more maintained fork

moni-dz committed:
linuxPackages.rtl88x2bu: switch to more maintained fork

Commit:
use version number as rev for git source

r-ryantm committed:
agi: 1.1.0-dev-20210514 -> 1.1.0-dev-20210517

r-ryantm committed:
bemenu: 0.5.0 -> 0.6.0

marsam committed:
ncspot: 0.7.0 -> 0.7.1

https://github.com/hrkfdn/ncspot/releases/tag/v0.7.1

r-ryantm committed:
caddy: 2.4.0 -> 2.4.1

r-ryantm committed:
delve: 1.6.0 -> 1.6.1

Lassulus committed:
Merge pull request #108348 from rencire/add-hid-nintendo

linuxPackages.hid-nintendo: init at 3.1

r-ryantm committed:
doppler: 3.24.4 -> 3.25.0

zowoq committed:
libslirp: 4.4.0 -> 4.5.0

https://gitlab.freedesktop.org/slirp/libslirp/-/releases/v4.5.0

zowoq committed:
slirp4netns: 1.1.9 -> 1.1.10

https://github.com/rootless-containers/slirp4netns/releases/tag/v1.1.10

stephank committed:
ffmpeg-full: fix darwin build

libglslang is a dependency of vulkan filters, so only makes sense if
those are enabled.

moni-dz committed:
linuxPackages.rtl8812au: add myself as maintainer, cleanup

r-ryantm committed:
gh: 1.10.1 -> 1.10.2

Lassulus committed:
Merge pull request #108138 from Lassulus/yarn2nix_move

yarn2nix: updates from nix-community/yarn2nix

jnetod committed:
pythonPackages.pysmf: fix build with python 3.9

evils committed:
element-desktop: set dbus default for firefox

fixes opening links in firefox-wayland on wayland

vcunat committed:
Merge #123775: thunderbird*: 78.10.1 -> 78.10.2

https://www.thunderbird.net/en-US/thunderbird/78.10.2/releasenotes/
https://www.mozilla.org/en-US/security/advisories/mfsa2021-22/

veprbl committed:
Merge pull request #123638 from veprbl/pr/libAfterImage_fixes

libAfterImage: unvendor dependencies

veprbl committed:
Merge pull request #123581 from veprbl/pr/bibutils_enable_darwin

bibutils: enable darwin

r-ryantm committed:
jackett: 0.18.15 -> 0.18.95

saschagrunert committed:
skopeo: 1.2.3 -> 1.3.0

Signed-off-by: Sascha Grunert <sgrunert@redhat.com>

Ma27 committed:
Merge pull request #123764 from r-ryantm/auto-update/bemenu

bemenu: 0.5.0 -> 0.6.0

dotlambda committed:
retroshare: drop (#123345)

It was marked as broken in 2018.

roberth committed:
Merge pull request #117275 from hercules-ci/nixosTest-remove-nixpkgs-commit-hash

nixosTest: Make system.nixos.revision constant

hrdinka committed:
Merge pull request #123598 from pschyska/master

nixos/nsd: make nsd-checkconf work when configuration contains keys (#118140)

Ma27 committed:
Merge pull request #123693 from mayflower/matrix-openssl

matrix-synapse: add openssl to buildInputs

Ma27 committed:
Merge pull request #123721 from fabaff/bump-ipinfo

ipinfo: 1.1.2 -> 1.1.4

Ma27 committed:
Merge pull request #123465 from FliegendeWurst/devede-ffmpeg4

devede: ffmpeg_3 -> ffmpeg

legendofmiracles committed:
nixos/terraria: open ports in the firewall

primeos committed:
chromiumBeta: 91.0.4472.57 -> 91.0.4472.69

Ma27 committed:
Merge pull request #123464 from FliegendeWurst/renpy-ffmpeg4

renpy: ffmpeg_3 -> ffmpeg

davidak committed:
Merge pull request #123752 from mweinelt/tts

tts: 0.0.13 -> 0.0.14

r-ryantm committed:
bind: 9.16.15 -> 9.16.16

laMudri committed:
cmusfm: 2020-07-23 -> 2021-05-19

Bug fix for https://github.com/Arkq/cmusfm/issues/40.
Add self as maintainer.

moni-dz committed:
ats-acc: init at 2018-10-21

06kellyjac committed:
boundary: 0.2.1 -> 0.2.2

Co-authored-by: Robert Hensing <robert@roberthensing.nl>

roberth committed:
Merge pull request #123781 from 06kellyjac/boundary

boundary: 0.2.1 -> 0.2.2

06kellyjac committed:
trivy: 0.18.1 -> 0.18.2

06kellyjac committed:
nerdctl: 0.8.1 -> 0.8.2

r-ryantm committed:
abcmidi: 2021.05.10 -> 2021.05.19 (#123792)

dotlambda committed:
gopro: use ffmpeg instead of ffmpeg_3

benneti committed:
rmapi: 0.0.13 -> 0.0.15 (#123672)

dotlambda committed:
nwjs: use ffmpeg instead of ffmpeg_3

oxalica committed:
mycrypto: 1.7.13 -> 1.7.16

dotlambda committed:
manim: use ffmpeg instead of ffmpeg_3

dotlambda committed:
python3Packages.zulip: 0.7.1 -> 0.8.0

dotlambda committed:
zulip-term: add notify-send to $PATH

roberth committed:
Merge pull request #123052 from xoe-labs/da-test-vm-innteractive-log-switch

nixos/testing: add interactive serial stdout logs switch and dim them

kevincox committed:
fx_cast_bridge: 0.0.7 -> 0.1.2 (#121963)

The prevision version was out-of-date and incompatible with the latest extension.

This also updates to build from source.

Right now we also ship the devDependencies of the bridge due to limitations in node2nix.

saschagrunert committed:
buildah: 1.20.1 -> 1.21.0

Signed-off-by: Sascha Grunert <sgrunert@redhat.com>

collares committed:
sageWithDoc: update sagedoc.py test for sphinx 3.5

Commit:
soju: init at 0.1.2

netcrns committed:
orca-c: git-2020-05-01 -> git-2021-02-13

orca-c: prefer main git repo over github mirror

orca-c: add netcrns to maintainers

globin committed:
Merge pull request #123790 from r-ryantm/auto-update/bind

bind: 9.16.15 -> 9.16.16

jtojnar committed:
Merge pull request #122642 from fitzgibbon/dev/init-gnome-inform7

gnome-inform7: init at unstable-2021-04-06

primeos committed:
Merge pull request #123787 from primeos/chromiumBeta

chromiumBeta: 91.0.4472.57 -> 91.0.4472.69

mjlbach committed:
pythonPackages.matrix-nio: 0.18.0 -> 0.18.1

toastal committed:
julia-mono: 0.037 -> 0.039

Ma27 committed:
Merge pull request #123475 from dotlambda/xscast-ffmpeg

xscast: use ffmpeg instead of ffmpeg_3

Profpatsch committed:
ultrastardx: really don’t use ffmpeg_3

Due to a merge error, usdx didn’t actually switch to ffmpeg 4.
This fixes that

mweinelt committed:
Merge pull request #123816 from Profpatsch/ultrastardx-really-use-ffmpeg4

ultrastardx: really don’t use ffmpeg_3

netcrns committed:
orca-c: use `postPatch` instead of overriding `patchPhase`

netcrns committed:
orca-c: explicitly call `preInstall` & `postInstall` hooks during `installPhase`

mweinelt committed:
Merge pull request #123643 from Lassulus/solanum

solanum: 2020-12-14 -> 2021-04-27

felixonmars committed:
Correct a typo in python-modules/mox3

jonringer committed:
liquidsoap: fix build, add missing zlib

symphorien committed:
Merge pull request #123813 from mjlbach/matrix_nio_0_18_1

pythonPackages.matrix-nio: 0.18.0 -> 0.18.1

Commit:
maintainers: add malvo

zimbatm committed:
Merge pull request #123744 from hercules-ci/init-ghostunnel

ghostunnel: init

symphorien committed:
Merge pull request #122859 from symphorien/kdeconnect-buttons

kdeconnect: fix empty buttons

symphorien committed:
Merge pull request #120622 from symphorien/duplicity-master

nixos/duplicity: enable to prevent backup from growing infinitely

symphorien committed:
Merge pull request #123621 from JamieMagee/nzbhydra

nzbhydra2: refactor

symphorien committed:
Merge pull request #123655 from evils/vnstat

vnstat: 2.6 -> 2.7

symphorien committed:
Merge pull request #123777 from evils/element-wayland

element-desktop: set dbus default for firefox

onsails committed:
polkadot: 0.9.1 -> 0.9.2

sophrosyne97 committed:
wt: 4.2.0 -> 4.5.0

mweinelt committed:
Merge pull request #123180 from zeri42/zod-version-bump

rnhmjoj committed:
monero-gui: 0.17.2.1 -> 0.17.2.2

lsix committed:
freeipmi: 1.6.7 -> 1.6.8

See https://lists.gnu.org/archive/html/info-gnu/2021-05/msg00013.html
for release information

rnhmjoj committed:
pdns-recursor: 4.4.3 -> 4.5.1

collares committed:
sageWithDoc: fix static asset deduplication

delroth committed:
libvdpau-va-gl: drop obsolete ffmpeg_3 dependency (#123757)

I couldn't find any reference as to why this dependency would be
required for 0.4.2. Digging into the changelog, it looks like 0.4.0
dropped the libswscale dependency, so this seems like it's now unneeded.

fabaff committed:
metasploit: 6.0.44 -> 6.0.45

rnhmjoj committed:
qutebrowser: 2.2.1 -> 2.2.2

futile committed:
rustup: 1.24.1 -> 1.24.2

rycee committed:
sd-switch: 0.2.1 -> 0.2.2

dotlambda committed:
libextractor: use ffmpeg instead of ffmpeg_3 (#123344)

thoughtpolice committed:
Merge pull request #123618 from r-ryantm/auto-update/cloudflared

cloudflared: 2021.5.6 -> 2021.5.7

moni-dz committed:
linuxPackages_5_12.rtl88xxau-aircrack: fix build

7c6f434c committed:
Merge pull request #123844 from lsix/freeipmi-1.6.8

freeipmi: 1.6.7 -> 1.6.8

happysalada committed:
rustup: nixpkgs-fmt

marsam committed:
Merge pull request #123707 from leotaku/update-netdata

netdata: 1.30.1 -> 1.31.0

Ekleog committed:
pull request template: cleanup (#117537)

Remove elements of the PR template that have a low signal/noise ratio,
and add one that I think would have a good signal/noise ratio.

-----
Remove:
Determined the impact on package closure size (by running `nix path-info
-S` before and after)
-----
Rationale:
This is rarely done in practice, and apart from for specific packages
this is usually not a good indicator of anything useful

It might make sense to re-introduce it with two holes to fill, but then
we would have to make a serious decision to never land without these two
numbers filled in or with too big a regression, because in practice this
box has been a no-op in many cases.

Maybe just integrating this check in nixpkgs-review would bring the most
benefit here?
-----

-----
Remove:
Ensured that relevant documentation is up to date
-----
Rationale:
This is fuzzy, “relevant documentation” is way too often hard to find
-----

-----
Add:
Added a release notes entry if the change is major or breaking
-----
Rationale:
This is way too often forgotten, and is also a self-contained easy task
-----

happysalada committed:
rustup: fix darwin build

marsam committed:
Merge pull request #123850 from marsam/update-ncspot

ncspot: 0.7.0 -> 0.7.1

rnhmjoj committed:
Merge pull request #123724 from johnae/qutebrowser-pipewire

qutebrowser: support for webrtc pipewire (needed for wayland native screen sharing support)

rnhmjoj committed:
Merge pull request #123837 from rnhmjoj/qute

qutebrowser: 2.2.1 -> 2.2.2

danieldk committed:
Merge pull request #123515 from savannidgerinel/savanni/1password-8.0.34-stable

_1password-gui: 8.0.33-53.BETA -> 8.0.34-stable

onny committed:
Merge branch 'dokuwiki' of github.com:onny/nixpkgs into dokuwiki

onny commented:
ups sorry, I somehow messed up git rebase :(

1000101 commented:
> ups sorry, I somehow messed up git rebase :(

That's alright, you could have just force push the fix, no need to close whole PR.
